tests/defences/detector/poison/test_roni.py

```diff
@@ -1,6 +1,6 @@
 # MIT License
 #
-# Copyright (C) The Adversarial Robustness Toolbox (ART) Authors 2018
```

Comment:
I think this was initially implemented in 2018.

Reply:
Fixed in next commit